test/ng/compileSpec.js

```diff
@@ -6327,6 +6330,54 @@ describe('$compile', function() {
       });
     });
 
+    describe('one-way collection bindings', function() {
+      it('should update isolate scope when origin scope changes', inject(function() {
+        $rootScope.collection = [{
+          name: 'Gabriel',
+          value: 18
+        }, {
+          name: 'Tony',
+          value: 91
+        }];
+        $rootScope.query = '';
+        $rootScope.$apply();
+
+        compile('<div><span my-component ow-colref="collection | filter:query" $ow-colref$="collection | filter:query">');
+
+        expect(componentScope.owColref).toEqual($rootScope.collection);
+        expect(componentScope.owColrefAlias).toEqual(componentScope.owColref);
+        expect(componentScope.$owColrefAlias).toEqual(componentScope.owColref);
+
+        $rootScope.query = 'Gab';
+        $rootScope.$apply();
+
+        expect(componentScope.owColref).toEqual([$rootScope.collection[0]]);
+        expect(componentScope.owColrefAlias).toEqual([$rootScope.collection[0]]);
+        expect(componentScope.$owColrefAlias).toEqual([$rootScope.collection[0]]);
+      }));
+
+      it('should update origin scope when isolate scope changes', inject(function() {
+        $rootScope.collection = [{
+          name: 'Gabriel',
+          value: 18
+        }, {
+          name: 'Tony',
+          value: 91
+        }];
+
+        compile('<div><span my-component ow-colref="collection">');
+
+        var newItem = {
+          name: 'Pablo',
+          value: 10
+        };
+        componentScope.owColref.push(newItem);
+        componentScope.$apply();
+
+        expect($rootScope.collection[2]).toEqual(newItem);
+      }));
+    });
```

Comment:
Missing a test to verify the `$watchCollection` part. (AFAICT, these tests would have passed even if `*` was ignored.)

Reply:
I think the first test verifies it, and this test is just wrong/not-applicable (you can tell I copied them both from the `=*` tests...).